lib/iris/coords.py

```diff
@@ -1478,32 +1619,74 @@ def _repr_other_metadata(self):
             result += ', circular=%r' % self.circular
         return result
 
-    @property
-    def points(self):
-        """The local points values as a read-only NumPy array."""
-        return self._points
-
-    @points.setter
-    def points(self, points):
-        points = np.array(points, ndmin=1)
```

Comment:
Deliberately! We want new functionality (i.e. `as_concrete_data` and the `DataManager.data` setter) to ensure we have a real array rather than injecting a random call to a NumPy function...